source/CollisionSet.cpp

```diff
@@ -254,9 +274,12 @@ Body *CollisionSet::Line(const Point &from, const Point &to, double *closestHit,
 			if(it->x != gx || it->y != gy)
 				continue;
 
-			if(seen.count(it->body))
+			bool present = any_of(seen.begin(), seen.end(), [it](const Body *el) {
```

Comment:
This is basically a find, so if you replace this with a call to `std::find` you don't need the lambda anymore.

Reply:
Solved